src/test/scala/akka/persistence/cassandra/journal/PubSubThrottlerSpec.scala

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright (C) 2016 Typesafe Inc. <http://www.typesafe.com>
+ */
+package akka.persistence.cassandra.journal
+
+import scala.concurrent.duration.DurationInt
+
+import org.scalatest.{ MustMatchers, WordSpecLike }
+
+import akka.actor.{ ActorSystem, Props }
+import akka.testkit.{ TestKit, TestProbe }
+
+class PubSubThrottlerSpec extends TestKit(ActorSystem("CassandraConfigCheckerSpec"))
+  with WordSpecLike with MustMatchers {
+  "PubSubThrottler" should {
+    "eat up duplicate messages that arrive within the same [interval] window" in {
+      val delegate = TestProbe()
+      val throttler = system.actorOf(Props(new PubSubThrottler(delegate.ref, 100.milliseconds)))
+
+      throttler ! "hello"
+      throttler ! "hello"
+      throttler ! "hello"
+      delegate.within(10.milliseconds) {
+        delegate.expectMsg("hello")
```

Comment:
we can't use this tight timeouts, just a tiny GC at the wrong time will fail this test, don't use shorter than 2 seconds